ui-toolkit/modals.md

```diff
@@ -1,7 +1,148 @@
 ---
 layout: page
-title:  Modals
+title:  Modal windows
 category: UI toolkit
 ---
 
-Content here!
\ No newline at end of file
+- [Use cases](#use)
+- [Behavior](#behavior)
+- [Content guidelines](#guidelines)
+- [Style](#style)
+- [Code](#code) 
+{: class="toc"}
+
+<p>Modal windows (also known as dialog boxes or lightboxes) are “pop-up” elements that appear in front of a web page, blocking the main page below. Similar to popup windows or system alerts, modals are unique because they prevent interaction with the page underneath. </p> {: class="lead-in"}
+
+<h2 id="use">Use cases<span class="cf-code-link">
+</span></h2>
+
+<p>Modals help draw attention to an alert or a focused task by preventing interactions with anything on the web page other than the modal itself. Due to their disruptive nature, modal windows should only be used in very specific cases outlined below.</p>
+
+<h3 id="feedback-correction">Feedback or correction</h3>
+
+<p>Modals can be effective for communicating a warning or alert outside the main web page. They can be triggered by a user interaction (like pressing a button to “save”) or without an interaction (like a timeout warning after a period of inactivity).</p>
+
+<h3 id="interruption">Interruption</h3>
+<p>Use modals to force the user to complete a task or make a decision that requires their full attention outside of the main workflow. A critical piece of this scenario is that you need the workflow to be interrupted to be effective. If not, explore other UI elements to accomplish the goal.</p> 
+
+<div class="content-50 content-first">
+  <h4 id="examples">Examples of when to use modals</h4>
+  <ul>
+    <li>Confirm an action that can’t be undone, such as “Are you sure you want to delete this?</li>
```

Comment:
Also, we'll probably want to code all the curly quotation marks as either `&ldquo;` for the opening one or `&rdquo;` for the closing ones.